benchmarks/nemo/nemolite2d/manual_versions/psykal_opencl/xilinx.cfg

```diff
@@ -0,0 +1,38 @@
+# Create 2 CU of the given kernels
```

Comment:
Please add a comment describing the purpose of this file.

Reply:
Added comment